lacommunaute/notification/tests/tests_tasks.py

```diff
@@ -68,6 +72,41 @@ def test_send_notifs_when_first_reply(self):
         self.assertEqual(email_sent_track.datas, payload)
         self.assertEqual(email_sent_track.kind, "first_reply")
 
+    @respx.mock
+    def test_send_notifications_asap(self):
+        topic = TopicFactory(with_post=True)
+        notif_asap = NotificationFactory(post=topic.first_post, delay=NotificationDelay.ASAP)
+        notif_day = NotificationFactory(post=topic.first_post, delay=NotificationDelay.DAY)
+
+        def get_expected_payload_for_notification(notification):
+            params = {
+                "poster": notification.post.poster_display_name,
+                "action": f"a répondu à '{notification.post.subject}'",
+                "forum": notification.post.topic.forum.name,
+                "url": notification.post.topic.get_absolute_url(with_fqdn=True),
+            }
+
+            return {
+                "to": [{"email": DEFAULT_FROM_EMAIL}],
```

Comment:
note to self : `DEFAULT_FROM_EMAIL` en destinataire provoque une erreur dans Brevo